### PR TITLE
docs: extended security guide to include OpenSSF tooling

### PR DESCRIPTION
**Pull request by UlisesGascon**
Hi all!

I am not very sure about how to manage the translations with the latests changes, but I am happy to update the spanish version for this change as well in this PR or in a separate one.

Regarding the pure content changes, let's see if there are at least 2 more members of the @nodejs/security that can review it before we merge it. 😉

We discuss to include this references in https://github.com/nodejs/security-wg/issues/932

### CODE REVIEW

**Review by ovflowd — APPROVED** [review]
Looks good to me :)

For translations, check `TRANSLATION.md`!

**Comment by ovflowd** [comment]
Hey team, can we `fast-track` this PR? @nodejs/website

**Review by RafaelGSS — APPROVED** [review]

**Review by aymen94 — APPROVED** [review]

File: pages/en/docs/guides/security/index.md [hunk 1]
```diff
@@ -410,6 +410,13 @@ The use of experimental features in production isn't recommended.
 Experimental features can suffer breaking changes if needed, and their
 functionality isn't securely stable. Although, feedback is highly appreciated.
 
+## OpenSSF Tools
+
+The [OpenSSF][] is leading several initiatives that can be very useful, especially if you plan to publish an Npm package. These initiatives include:
```
Comment (ghost) [inline pages/en/docs/guides/security/index.md]: Maybe 'Npm' to be better in 'npm' ("N" in a lower case more natural)
or just use ```npm```?

**Review by ghost — APPROVED** [review]

File: pages/en/docs/guides/security/index.md (hunk 1 above)
Comment (ljharb) [reply pages/en/docs/guides/security/index.md]: branding-wise, it’s always npm, lowercase, and never Npm for sure.

Comment (shanpriyan) [inline pages/en/docs/guides/security/index.md]:
```suggestion
The [OpenSSF][] is leading several initiatives that can be very useful, especially if you plan to publish an npm package. These initiatives include:
```

**Review by shanpriyan — APPROVED** [review]
LGTM :shipit:

**Comment by ovflowd** [comment]
We need one more sign-off from @nodejs/security

**Review by manishprivet — APPROVED** [review]

**Comment by RafaelGSS** [comment]
> We need one more sign-off

That's fine. I think we can land it.

**Pull request merged**